source/sync.txt

```diff
@@ -8,21 +12,129 @@
    :titlesonly:
    :hidden:
    
-   What is {+sync+}? </sync/learn>
+   Get Started </sync/get-started>
    Configure and Update Your Data Model </sync/configure-your-data-model>
    Define Data Access Patterns </sync/data-access-patterns>
    Configure Sync </sync/configure-device-sync>
    Handle Errors </sync/error-handling>
    Add Sync to a Local-Only App </sync/local-to-sync>
+   Production Checklist </sync/production-checklist>
    Sync Production Load Testing </sync/production-load-testing>
    Event Library </sync/event-library>
+   Technical Details </sync/details>
+
+.. introduction::
+
+   You're developing a mobile app. Your users want their data saved on the cloud
+   and accessible from their other devices. Network access on a mobile device
+   can be hit or miss, so you write data locally on the device first. A
```

Comment:
Nit: hit or miss is an idiom that may be unfamiliar to folks from other countries or for whom English is a second language. What about an alternative wording?

```suggestion
   can be intermittent, so you write data locally on the device first. A
```